docs/modules/record-validation/proc-record-validation.adoc

```diff
@@ -0,0 +1,83 @@
+// file included in the following:
+//
+// assembly-record-validation-filter.adoc
+
+[id='proc-record-validation-{context}']
+= (Preview) Setting up the Record Validation filter
```

Comment:
We need an abstract for the procedure, like the Record Encryption filter.  "This procedure describes how to set up the Record Validation filter. Provide the filter configuration and rules that the filter uses to check against Kafka record keys and values."

Reply:
done.